test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/VersionSpecificTests.cs

```diff
@@ -31,5 +34,12 @@ private async Task TestCapabilityStatementFhirVersion(string expectedVersion)
             Assert.NotNull(capabilityStatement.FhirVersionElement);
             Assert.Equal(expectedVersion, capabilityStatement.FhirVersionElement.ObjectValue);
         }
+
+        private void TestSupportedVersion(string expectedVersion)
```

Comment:
You should be able to do the `TestSupportedVersions` the same, but in the Shared.Core.Tests project. Reason is that testing `IModelInfoProvider` seems like a unit test. Nothing goes over the wire or requires a server for this

Reply:
Makes sense! Removed the `TestSupportedVersion` method and added new unit test classes to R4.Core.UnitTests and Stu3.Core.UnitTests in 7abc392.